index.html

```diff
@@ -6,6 +6,175 @@
+            </div>
+            <div>
+                <h4>Accessory makers</h4>
+                <p>While most people enjoy casino gambling, sports betting, lottery and bingo playing for the fun</p>
+            </div>
+            <div>
+                <h4>Alterationists</h4>
+                <p>If you are looking for a new way to promote your business that won’t cost you more money, </p>
+            </div>
+            <div>
+                <h4>Custom Design designers</h4>
+                <p>If you are looking for a new way to promote your business that won’t cost you more money, </p>
+            </div>
+            <div>
+                <img src="images/design prof.png" alt="a man with phone" srcset="">
+            </div>
+        </section>
+        <section>
+            <div>
+                <p>Easier decision making for</p>
+                <h3>Product Managers</h3>
+                <p>The Myspace page defines the individual, his or her characteristics, traits, personal choices and the overall personality of the person. </p>
+            </div>
+            <div>
+                <ul>
+                    <li>
+                        <img src="icons/Rectangle.svg" alt="Rectangle" srcset="">
+                        <p>Never worry about overpaying for your energy again. </p>
+                    </li>
+                    <li>
+                        <img src="icons/Rectangle.svg" alt="Rectangle" srcset="">
+                        <p>We will only switch you to energy companies that we trust and will treat you right</p>
+                    </li>
+                    <li>
+                        <img src="icons/Rectangle.svg" alt="Rectangle" srcset="">
+                        <p>We track the markets daily and know where the savings are.</p>
+                    </li>
+                </ul>
+            </div>
+            <div>
+                <img src="images/product managers.png" alt="a man with laptop" srcset="">
+            </div>
+        </section>
+        <section>
+            <div>
+                <p>Optimisation for</p>
+                <h2>Marketers</h2>
+                <p>Few would argue that, despite the advancements of feminism over the past three decades, women still face a double standard when it comes to their behavior. </p>
+            </div>
+            <div>
+                <h4>Accessory makers</h4>
+                <p>While most people enjoy casino gambling, sports betting, lottery and bingo playing for the fun</p>
+            </div>
+            <div>
+                <h4>Alterationists</h4>
+                <p>If you are looking for a new way to promote your business that won’t cost you more money.</p>
+            </div>
+            <div>
+                <h4>Custom Design designers</h4>
+                <p>If you are looking for a new way to promote your business that won’t cost you more money.</p>
+            </div>
```

Comment:
тут треба замінити `h4` => `h3`
```suggestion
            <div>
                <h3>Accessory makers</h3>
                <p>While most people enjoy casino gambling, sports betting, lottery and bingo playing for the fun</p>
            </div>
            <div>
                <h3>Alterationists</h3>
                <p>If you are looking for a new way to promote your business that won’t cost you more money.</p>
            </div>
            <div>
                <h3>Custom Design designers</h3>
                <p>If you are looking for a new way to promote your business that won’t cost you more money.</p>
            </div>
```